### PR TITLE
Update oralfacial_analysis and requirements

### PR DESCRIPTION
**Pull request by LiuDaveLiu**

### CODE REVIEW

**Comment by LiuDaveLiu** [comment]
@hanhou Are you fine with the changes in requirements.txt?

File: MATLAB/pipeline/+tracking/JawTracking3DSid.m [hunk 1]
```diff
@@ -11,11 +11,19 @@
     properties
         keySource = (experiment.getSchema().v.Session & 'rig = "RRig-MTL"')...
             & (tracking.getSchema().v.Tracking & 'tracking_device = "Camera 3"')...
-            & (tracking.getSchema().v.Tracking & 'tracking_device = "Camera 4"');
+            & (tracking.getSchema().v.Tracking & 'tracking_device = "Camera 4"')...
```
Comment (ttngu207) [inline MATLAB/pipeline/+tracking/JawTracking3DSid.m:14]: this line was removed and now added in, is this intentional, if so, perhaps you mean

```
& (tracking.getSchema().v.Tracking & 'tracking_device in ("Camera 3", "Camera 4")')...
```

tracking_device being either `Camera 3` or `Camera 4` ?

File: MATLAB/pipeline/+tracking/LickPortTracking3DBot.m [hunk 2]
```diff
@@ -11,12 +11,20 @@
     properties
         keySource = (experiment.getSchema().v.Session & 'rig = "RRig-MTL"')...
             & (tracking.getSchema().v.Tracking & 'tracking_device = "Camera 3"')...
-            & (tracking.getSchema().v.Tracking & 'tracking_device = "Camera 4"');
+            & (tracking.getSchema().v.Tracking & 'tracking_device = "Camera 4"')...
```
Comment (ttngu207) [inline MATLAB/pipeline/+tracking/LickPortTracking3DBot.m:14]: same comment as above

File: MATLAB/pipeline/+tracking/JawTracking3DSid.m (hunk 1 above)
Comment (LiuDaveLiu) [reply MATLAB/pipeline/+tracking/JawTracking3DSid.m:14]: No, this was not intentional, I meant it must have tracking in Camera 3 AND Camera 4

**Review by ttngu207 — APPROVED** [review]

**Pull request merged**